simd/src/Kokkos_SIMD_AVX2.hpp

```diff
@@ -925,6 +974,40 @@ simd<std::int64_t, simd_abi::avx2_fixed_size<4>>::simd(
       _mm256_sub_epi64(static_cast<__m256i>(lhs), static_cast<__m256i>(rhs)));
 }
 
+KOKKOS_IMPL_HOST_FORCEINLINE_FUNCTION
+simd<std::uint64_t, simd_abi::avx2_fixed_size<4>> operator>>(
+    simd<std::uint64_t, simd_abi::avx2_fixed_size<4>> const& lhs,
+    int rhs) noexcept {
+  return simd<std::uint64_t, simd_abi::avx2_fixed_size<4>>(
+      _mm256_srli_epi64(static_cast<__m256i>(lhs), rhs));
+}
+
+KOKKOS_IMPL_HOST_FORCEINLINE_FUNCTION
+simd<std::uint64_t, simd_abi::avx2_fixed_size<4>> operator>>(
+    simd<std::uint64_t, simd_abi::avx2_fixed_size<4>> const& lhs,
+    simd<std::int32_t, simd_abi::avx2_fixed_size<4>> const& rhs) noexcept {
+  return simd<std::uint64_t, simd_abi::avx2_fixed_size<4>>(
+      _mm256_srlv_epi64(static_cast<__m256i>(lhs),
+                        _mm256_cvtepi32_epi64(static_cast<__m128i>(rhs))));
+}
```

Comment:
same question as above

Reply:
Changed in [63c8055](https://github.com/kokkos/kokkos/pull/6109/commits/63c80558527bb520f6b2fb6c5ae605c94b8c362d).